asm/battle_1.s

```diff
@@ -1424,8 +1424,8 @@ _0803575A:
 	.pool
 	thumb_func_end ApplyPlayerChosenFrameToBattleMenu
 
-	thumb_func_start load_battle_oval_graphics
-load_battle_oval_graphics: @ 8035770
+	thumb_func_start LoadFittingBackgroundForBattle
```

Comment:
What's meant by "fitting" here?

Reply:
If battle flag groudon -> loads groudon background
if battle flag kyogre -> loads kyogre background
same with rayquaza
then checks if the trainer class is leader/champion
then either loads it from gBattleTerrain or one that fits the map background

Reply:
So then seems like it should just be called "LoadBackgroundForBattle".

Reply:
Makes sense I guess.